nixos/modules/hardware/logiops/default.nix

```diff
@@ -0,0 +1,79 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.logiops;
+  renderedConfig =
+    (pkgs.formats.libconfig { }).generate "logid.cfg" cfg.settings;
```

Comment:
```suggestion
  renderedConfig = (pkgs.formats.libconfig { }).generate "logid.cfg" cfg.settings;
```

Reply:
nixpkgs-fmt doesn't change this and it *is* a long line. Why?

Reply:
because nixpkgs-fmt is not black and won't change long lines back.

Reply:
It's also over 80 columns and won't fit on my whole screen if I have another window open next to it which is a bummer.

(Anyhow, "fixed")